src/main/scala/com/wavesplatform/state2/reader/StateReaderImpl.scala

```diff
@@ -76,4 +72,10 @@ class StateReaderImpl(p: StateStorage) extends StateReader {
     .map(_.getKey)
     .map(EqByteArray)
     .toSeq
```

Comment:
While we're at it, please remove `entrySet` and replace filter/map with collect.